### ISSUE TITLE
The body content is not replaced (subject, description).

### ISSUE THREAD

**Issue by sgithubs** [akiko-pusu/redmine_issue_templates#73]
When I change tracker, for example from "Bug" to "Task" (I have 2 different templates for "Bug" and for "Task"), the body content which was filled is not replaced. I can replace content just after clicking on "Clear subject and description text".
For example, I use "Bug" template by default, but if user want to change from "Bug" to "Task" he need to click "Clear subject and description text" button. Plugin should change body content automatically, when I change templates. 
It is very uncomfortable for us.
It you fix it - it will be great!
Thanks!

**Comment by jwciss** [comment]
The current behavior should not be changed. We use multiple different global templates for different trackers. Once the tracker is changed, it would be quite problematic if the content were automatically changed. Especially if a user has already entered some text an wants to change the tracker afterwards.

**Comment by sgithubs** [comment]
Just a remark: I don`t use global templates.

And about (user has already entered some text and wants to change the tracker afterwards):

We have different templates for different trackers. First of all, user should check Tracker correctly, then template for this tracker, then description (according to template).
If user wants to change Tracker, the template will change too and user should enter new text according to this template.

**Comment by akiko-pusu** [comment]
Hi, @sgithubs 
Thanks for your creating request.  
Have you tried "Replace subject and description" option of template at project module tab?
Is it enough?
Or do you hope overwrite description content just only after changing tracker?

**Comment by akiko-pusu** [comment]
@sgithubs @jwciss 
Thanks for your  comments on this topic.
I’ve recently released related feature above topic.

1. Enabled to mark as default Global issue template. 
2. Add feature to revert title and description just after template applied (Both happened and overwrite)
3. Add feature that popup confirmation dialog when template intends to overwrite title and description.

I hope these would be some of help. Now you can try these update on master branch.

**Comment by akiko-pusu** [comment]
How have things been after that?
I close this issue, so please post new issue if you have any other trouble.
